Contributors.md

```diff
@@ -98,3 +98,4 @@ Thank you everyone that have contributed to creating this awesome guide!
 - [sslees](https://github.com/sslees)
 - [woutercx](https://github.com/woutercx)
 - [paicha](https://github.com/paicha)
+- [rohitjmathew](https://github.com/rohitjmathew)
```

Comment:
This happens automatically on every deploy so no need to do it manually (for future reference)